src/app/compare/compare-tree/compare-tree.component.ts

```diff
@@ -40,15 +42,19 @@ export class CompareTreeComponent {
     highlightOpenFolders: false,
     folderBehaviourOnClick: 'select',
     expandAllFolders: true,
+    determineIconClass: SimpleFileTreeUtil.conditionalCssClass,
   };
 
   rightTreeOptions: FileTreeOptions = {
     highlightOpenFolders: false,
     folderBehaviourOnClick: 'select',
     expandAllFolders: true,
     determineFontColor: (item: CreateTreeItem) => this.setRedLabels(item),
+    determineIconClass: SimpleFileTreeUtil.conditionalCssClass,
```

Comment:
See other comment

Reply:
No the method is passed as a reference here